home/templates/home/prerelease/event_list.html

```diff
@@ -27,10 +27,29 @@ <h3>Events</h3>
             {% endif %}
             </div>
         </div>
+        <h2>Upcoming Events</h2>
         <div class="g-2 g-lg-3 row row-cols-4 justify-content-evenly pt-4">
 
-            {% for event in events %}
+            {% for event in upcoming_events %}
             {% include 'home/includes/event_card.html' %}
+            {% empty %}
+            <div class="text-center">
+                <img alt="No data image" class="no-data-img" src="{% static 'img/no_data.svg' %}" />
+                <p class="pt-4">No upcoming events.</p>
```

Comment:
```suggestion
                <p class="pt-4 pb-4">No upcoming events.</p>
```
On mobile, it's too close to the next title without padding